node/rest/src/routes.rs

```diff
@@ -338,9 +338,27 @@ impl<N: Network, C: ConsensusStorage<N>, R: Routing<N>> Rest<N, C, R> {
         Json(solution): Json<Solution<N>>,
     ) -> Result<ErasedJson, RestError> {
         // If the consensus module is enabled, add the unconfirmed solution to the memory pool.
-        if let Some(consensus) = rest.consensus {
+        // Otherwise, verify it prior to broadcasting.
```

Comment:
I can imagine spamming the REST endpoint with solutions could halt a client due to the amount of verifications required. But we should test (cc @iamalwaysuncomfortable).

If it turns out to be an issue, we can:
- add a separate [solutions rate limit](https://docs.rs/tower_governor/0.3.2/tower_governor/governor/struct.GovernorConfigBuilder.html#method.methods) using a second `.layer(GovernorLayer { }`.
- queue solutions to verify, takes more work but seems more robust